openvpn/start.sh

```diff
@@ -78,4 +78,15 @@ if [ -n "${LOCAL_NETWORK-}" ]; then
   fi
 fi
 
+
+if [ "${WEBPROXY_ENABLED}" = "true" ]; then
+  if [ -z "$WEBPROXY_PORT" ] ; then
+    /opt/tinyproxy/setport.sh $WEBPROXY_PORT
```

Comment:
This whole addition needs to be moved to the new up script. And in addition we need a section [like this](https://github.com/jandrews377/docker-transmission-openvpn/blob/0877052a13315791a8ece76d3036f36567db17e1/openvpn/start.sh#L74) to open port in UFW if both UFW and proxy is enabled.